crepecake/src/integration-test/java/com/google/cloud/tools/crepecake/registry/LocalRegistry.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2018 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.google.cloud.tools.crepecake.registry;
+
+import java.io.IOException;
+import org.junit.rules.ExternalResource;
+import org.junit.rules.TestRule;
+
+/** {@link TestRule} that runs a local registry. */
+class LocalRegistry extends ExternalResource {
```

Comment:
Can we customize the registry's storage location? so that if "STOP" fails it will still be in a unique temp location?

Reply:
Made each registry be a unique container.